README.md

```diff
@@ -21,55 +21,60 @@ The **rticles** package provides a suite of custom [R Markdown](http://rmarkdown
 
 - [ACM](http://www.acm.org/) articles
 
-- [ACS](http://pubs.acs.org/) articles
+- [ACS](http://pubs.acs.org/) articles (contributed by @yufree)
```

Comment:
I forgot to mention that besides the full journal names, it might be a good idea to also add the output format name in the list like:

```suggestion
- `rticles::acs_article`: [ACS](http://pubs.acs.org/) articles (contributed by @yufree)
```

Reply:
I add the same idea before I see this comment but I put it at the end without `rticles::`
https://github.com/rstudio/rticles/pull/304/commits/62cfd1c191736d4255ebd1d91c3ed09e3559f7c9

Do you prefer it at the beggining as you show here ?

Reply:
I don't have a preference. It's fine to put it at the end.